x/supply/types/key.go

```diff
@@ -0,0 +1,5 @@
+package types
+
+const (
+	ModuleName = "supplyqueries"
```

Comment:
`supplyqueries` is odd. Can we just use `supply` directly?

```suggestion
	ModuleName = "supply"
```

Reply:
Looks like we can't use `supply`, how about `akashsupply`?